### ISSUE TITLE
Git version 2.0.0 is required. Found 1.8.5.

### ISSUE THREAD

**Issue by adyz** [semantic-release/semantic-release#856]
## Current behavior
```
[Semantic release]: Running semantic-release version 15.6.1
[semantic-release]: Git version 2.0.0 is required. Found 1.8.5.
npm ERR! code ELIFECYCLE

npm ERR! errno 1

npm ERR! @xxxx-components@0.0.0-development semantic-release: `npm run build && semantic-release && npm run deploy-storybook -- --ci`

npm ERR! Exit status 1

npm ERR! 

npm ERR! Failed at the @xx/xx-components@0.0.0-development semantic-release script.

npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Is there any way to make this work on older versions of git? I don't have the technical capabilities to update the git version.

**Comment by adyz** [comment]
Using an older version of semantic-release works, but what would be the safest version to use that dose not require git version greater than 1.8.5?
Any thoughts?


Thank you!

**Comment by pvdlg** [comment]
Older version of semantic-release didn't supported version < 2.0.0 either. It's that we were not checking for the git version. Not all the git command we do require 2.0.0 so depending on what semantic-release is doing it might or might not use a command that require 2.0.0.

I'm not 100% sure what git version is required and what command require what, because there is no good way to know what version of Git support which options/command. The only way is too look at the the [Git doc](https://git-scm.com/docs) and use the `version` dropdown at the top of each page to select an older version and see if the needed option/command was available in said version. However the oldest version available in the doc is 2.0.0.

The version [2.0.0](https://github.com/git/git/releases/tag/v2.0.0) has been released on May 28, 2014, more than 4 years ago, so I think it's more than reasonable to require that version. In addition semantic-release is a development support tool, not a production dependency, so we think it's ok to require updated tools. This is why we [require Node 8.3](https://github.com/semantic-release/semantic-release/blob/caribou/docs/support/node-version.md) for example.

Updating to a new version is fairly easy and a good thing to do anyway. In any case it's a lot easier than accepting older version in semantic-release and find out what is supported or not in an undocumented 4 years+ old version. Going through the commit history of [git](https://github.com/git/git) and analyze the code in each version to figure out what is supported or not is extremely painful and time consuming. This time is way better invested in fixing bugs and developing features.

**Comment by adyz** [comment]
Thank you for your answer @pvdlg !
You're right, we should update our git version.

* I've installed the semantic-release 15.0.0 and looks like this message doesn't show up.
